### PR TITLE
Add work + aboutme sections

### PR DESCRIPTION
**Pull request by jaferIdrees**
I have added the below sections:
- My recent works section
- About me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,51 +11,183 @@
...
+        <li> <a href="#" class="smIcom"><img src="images/github.svg" alt=""></a> </li>
+        <li> <a href="#" class="smIcom"><img src="images/angellist.svg" alt=""></a> </li>
+      </ul>
+      <div class="nextPageIcon">
+        <a href=""><img src="images/Disabled.svg" alt=""></a>
+      </div>
+    </section>
+  </div>
+  <div class="workTitle">
+    <h1 id="h1WorkTitle">My Recent Works</h1>
```
Comment (m05e5) [inline index.html]: - [ ] please its recommenced to follow HTML semantic code. So you should only have 1 `h1` tag per file.

File: index.html [hunk 2]
```diff
@@ -11,51 +11,183 @@
...
+        <li class="lang langName"><a class="lang langName" href="">Bootstrap</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Ruby on rails</a></li>
+        <li class="lang langName"><a class="lang langName" href="">RSpec</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Capybara</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Selenium</a></li>
+      </ul>
+    </div>
+    <div class="cardAM">
+      <img src="images/icon-skills.svg" alt="" class="imageAmCard">
+      <h1 class="cardAMTitle">Skills</h1>
```
Comment (m05e5) [inline index.html]: same here please change to a `hx` other than `h1`

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @jaferIdrees , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] great job but please kindly stick to the design for the button text color
**figma**
image
**imp**
image


_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (zieeco) [reply index.html]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html (hunk 2 above)
Comment (zieeco) [reply index.html]: - [x] This issue is well resolved :heavy_check_mark:

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @jaferIdrees 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._

- [ ] As required by the previous reviewer, please let's stick to the design as much as we can here:

**your button**

design

**figma design**

b2

- kindly adjust your styling to match the design.





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -11,51 +11,183 @@
...
+    </section>
+  </div>
+  <div class="workTitle">
+    <h2 id="h1WorkTitle">My Recent Works</h2>
+    <img src="images/Indicator.svg" alt="">
+  </div>
+  <section class="gridContainer">
+    <div class="gridCell works">
+      <article class="cardContainer">
+        <img src="images/Img_Plaholder.svg" alt="" class="imgPlaceholder">
```
Comment (omar-muhamad) [inline index.html:71]: - [ ] [Optional] Please add the correct image height to all project card images to match the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @jaferIdrees,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### ⚠️Please consider doing the Optional changes mentioned below I marked it as optional because the last reviewer forgot to mention it.⚠️



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**